src/nbb/impl/repl.cljs

```diff
@@ -189,6 +190,7 @@
    'socket-repl (sci/copy-var socket-repl rns)})
 
 (defn init []
+  (println (str "Welcome to nbb v" (api/version) "."))
```

Comment:
I think a better place for this println would be in the `repl` function above.

Reply:
Yes for sure. Will do.

Reply:
Done